parsers/MK_wrapper.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime, timedelta
+from logging import Logger, getLogger
+from typing import List, Optional, Union
+
+from requests import Session
+
+from . import ENTSOE
+from .lib.config import refetch_frequency
+
+"""
+This wrapper is used to fetch data from ENTSOE for the country of North Macedonia.
+If no target_datetime is provided, the wrapper will provide a target datetime that is
+24 hours in the past due to intermittent data availability.
+"""
+
+
+def modify_target_datetime(target_datetime: Optional[datetime]) -> datetime:
+    if target_datetime is None:
+        return datetime.utcnow() - timedelta(hours=24)
```

Comment:
https://blog.ganssle.io/articles/2019/11/utcnow.html 
```suggestion
        return datetime.now(tz=utc) - timedelta(hours=24)
```

the utc timezone mus either come from datetime.timezone or pytz.utc